.github/workflows/push-deploy.yml

```diff
@@ -0,0 +1,26 @@
+name: Deploy to WordPress.org
+
+on:
+  workflow_dispatch:
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3
+
+      - name: Install Composer Dependencies
+        run: composer install --no-dev
```

Comment:
What do you think of changing this to: `composer install --no-dev --ignore-platform-reqs --prefer-dist --optimize-autoloader --no-progress`

Reply:
Yep, updated.